Automattic/jetpack
# Search: Update upsell page for pricing change

Fixes #25915 

#### Changes proposed in this Pull Request:

* Expose the `offset` option of the `IconTooltip` component to the Popover component and fix the relative position
* Modify `PricingTable` component styles adapting to the design and refactor the type of `tooltipInfo` to ReactNode for HTML syntax
* Introduce PricingTable to complete the upsell page for the new pricing change
-------------------------------------- updated --------------------------------------
* Introduce the URL parameter `new_pricing_202208=1` to determine when the new pricing table shows.
#### Other information:


#### Jetpack product discussion


p1HpG7-hEH-p2

#### Does this pull request change what data or activity we track or use?


N/A

#### Testing instructions:






* Go to a site with `Jetpack` connected but did not purchase `Jetpack Search`
* Navigate to the url `/wp-admin/admin.php?page=jetpack-search`
* Ensure there is the existing upsell page with a pricing card
* Navigate to the url `/wp-admin/admin.php?page=jetpack-search&new_pricing_202208=1`
* Ensure there is a brand new upsell page

| Before | After |
| --- | ----------- |
| <img width="1197" alt="截圖 2022-09-24 上午5 05 00" src="https://user-images.githubusercontent.com/6869813/192056720-c9656164-c962-4ade-b521-93aa54cf6046.png"> | <img width="1420" alt="截圖 2022-09-24 上午5 22 08" src="https://user-images.githubusercontent.com/6869813/192058492-3220a7ec-33c0-450a-bbe6-1f1cf4298a06.png"> |

* Ensure the new upsell page conforms to the design
<img width="924" alt="截圖 2022-09-24 上午5 08 32" src="https://user-images.githubusercontent.com/6869813/192057130-cbd4eb5b-1f8f-4a5f-8376-3372e3eeb78b.png">
